plugins/frontend/backstage-plugin-argo-cd/src/components/useAppDetails.ts

```diff
@@ -41,6 +41,9 @@ export const useAppDetails = ({
   ) => {
     const promises: Promise<void>[] | undefined =
       appDetails.status?.history?.map(async (historyRecord: any) => {
+        if (historyRecord.source?.chart !== undefined) {
```

Comment:
From the argo swagger docs for getting an application (`/api/v1/applications/{name}`) for `status.history[].source.helm: 

> Chart is a Helm chart name, and must be specified for applications sourced from a Helm repo.

Reply:
if this is defined revision will be a helm chart version and not git sha